recipe/meta.yaml

```diff
@@ -22,8 +22,9 @@ requirements:
     - python
     - matplotlib-base
     - numpy
-    - pillow >=5.3.0
+    - pillow >=5.3.0, <7
```

Comment:
ah, you don't support pillow 7???

Reply:
It was originally because of incompatibility between matplotlib 3.1.2 vs pillow 7. https://github.com/matplotlib/matplotlib/pull/16091
But it seems matplotlib 3.1.3 is already uploaded, maybe I can just remove this condition (ie <7)